.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Is this really necessary?

Reply:
I suppose not but it is helpful to me. In any case, I will revert that change.

Reply:
you can always use global git settings, for stuff that useful "for you"

Reply:
e.g.; `~/.gitignore_global`

Reply:
Good point!